apps/web/modules/insights/insights-view.tsx

```diff
@@ -51,12 +51,7 @@ export default function InsightsPage() {
 
   return (
     <div>
-      <Shell
-        withoutMain={false}
-        heading="Insights"
-        subtitle={t("insights_subtitle")}
```

Comment:
Can be removed because `title` takes precedence over `heading` and `description` over `subtitle`